src/sentry/apidocs/public_exclusion_list.py

```diff
@@ -733,6 +736,7 @@
     OrganizationEventsSpanOpsEndpoint,
     OrganizationEventsSpansExamplesEndpoint,
     OrganizationEventsSpansPerformanceEndpoint,
+    OrganizationEventsSpansHistogramEndpoint,
```

Comment:
mind removing this change and setting `private = True` on the endpoint? Will be removing this file soon.

Reply:
oh sorry, I didn't see the comment at the top of the file. Done 🙏